### PR TITLE
docs: add project configurations

### PR DESCRIPTION
**Pull request by stefanicjuraj**
- added project configurations option
- fixes https://github.com/daytonaio/docs/issues/98

This is a draft PR containing changes relating to adding the project configuration. I have added the basic information and commands for project configuration within the workspaces section - let me know if this should be changed to another section or a separate section should be created.

There will be additional commits containing improvements and rewrites.

### CODE REVIEW

File: src/content/docs/usage/workspaces.mdx [hunk 1]
```diff
@@ -319,3 +319,223 @@ daytona forward <PORT> --public
...
 
 </Aside>
+
+## Projects
+
+Projects represent a comprehensive set of settings and resources that define how your code is managed, built, and deployed within a Workspace. Each Project is associated with a specific repository and contains the necessary configurations to ensure a consistent development environment.
+
+## Project Configuration
+
+Project Configuration is designed to simplify the management of development environments, allowing you to predefine settings that can be easily applied to any new or existing Workspace.
```
Comment (Tpuljak) [inline src/content/docs/usage/workspaces.mdx:329]: `A Project Configuration`? (One line below as well)

**Comment by Tpuljak** [comment]
> thinking out loud, I think Project can be their own thing USAGE.
> 
> as you dont need to think about Projects if you dont want but can if you needed them. so might start off what they are and when would you use them.

When I read the comment, I didn't agree, but after reading the content of the PR, couldn't agree more 😄. Nice work @stefanicjuraj!

**Review by Tpuljak — CHANGES_REQUESTED** [review]

**Review by ivan-burazin — APPROVED** [review]

File: src/content/docs/usage/projects.mdx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  label: Projects
+---
+
+import Aside from '@components/Aside.astro'
+
+Projects represent a comprehensive set of settings and resources that define how your code is managed, built, and deployed within a Workspace. Each Project is associated with a specific repository and contains the necessary configurations to ensure a consistent development environment.
+
+## Project Configuration
+
+Project Configuration is designed to simplify the management of development environments, allowing you to predefine settings that can be easily applied to any new or existing Workspace.
```
Comment (Tpuljak) [inline src/content/docs/usage/projects.mdx:14]: `A ...` (2 lines below as well)

**Review by Tpuljak — CHANGES_REQUESTED** [review]

File: src/content/docs/usage/projects.mdx [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+Projects represent a comprehensive set of settings and resources that define how your code is managed, built, and deployed within a Workspace. Each Project is associated with a specific repository and contains the necessary configurations to ensure a consistent development environment.
+
+## Project Configuration
+
+A project configuration is designed to simplify the management of development environments, allowing you to predefine settings that can be easily applied to any new or existing Workspace.
+
+A project configuration contains all necessary properties to define and manage projects within a Workspace. It stores information such as the repository URL, build configuration, and environment variables. This allows for easy reuse of previously set configurations, ensuring a consistent and reproducible setup across multiple Workspaces.
+
+Project Configuration encapsulates all the essential elements for setting up a project, including:
```
Comment (Tpuljak) [inline src/content/docs/usage/projects.mdx:18]: `A` 😄

**Review by Tpuljak — CHANGES_REQUESTED** [review]

**Review by Tpuljak — APPROVED** [review]
Nice work!

**Pull request merged**